doc/source/user_guide/10min.rst

```diff
@@ -21,6 +21,14 @@ Object creation
 
 See the :ref:`Intro to data structures section <dsintro>`.
```

Comment:
This page contains a detailed explanation of what you're adding here. Probably better to have this after what you're writing, with something like `For more details about Series and DataFrame, see the intro to data structures section`.